Assets/Scripts/Game/Questing/Task.cs

```diff
@@ -95,6 +95,11 @@ public int GlobalVarLink
             get { return globalVarLink; }
         }
 
+        public List<IQuestAction> Actions
```

Comment:
Do you really need access to the list? I'm concerned with mods trying to add and remove from it. If you just need to read the actions, consider returning `IEnumerable<IQuestAction>` (no other change needed otherwise)

Reply:
I only need read access to the list, have updated that to return an `IEnumerable`, I have given it a quick re-test my end and works fine as expected.